next/src/assets/global.scss

```diff
@@ -85,16 +89,15 @@ main,
 .maxWidth {
   display: grid;
   --pageMargin: clamp(24px, calc(24vw/7.68), 100px);
-
-  width: calc(100% - var(--pageMargin)*2);
-  margin: 0 auto;
   grid-template-columns: 1fr;
 
 
 }
 
 .maxWidth {
   max-width: 1680px;
```

Comment:
Change the `max-width` to `1280px`, because the design probably will not look good in a larger viewport